### PR TITLE
Putting in additional sans for kubernetes.

### PR DESCRIPTION
**Pull request by mbruzek**
As it turns out kubernetes sets some static extra SANs

https://github.com/kubernetes/kubernetes/blob/master/cluster/images/hyperkube/static-pods/master-multi.json#L71

I put those in the reactive code and changed the tls layer to pull in the extra SANs only when creating the server certificate.

If anyone has other suggestions on how to do this I am really open to other approaches.

### CODE REVIEW

**Comment by mbruzek** [comment]
I built a charm with this code and deployed it to Amazon. I got the additional SANs in the server certificate.
X509v3 Subject Alternative Name: 
                IP Address:10.1.0.1, DNS:kubernetes, DNS:kubernetes.cluster.local, DNS:kubernetes.default, DNS:kubernetes.default.svc, DNS:kubernetes.default.svc.cluster.local, IP Address:54.67.44.203, IP Address:172.31.24.120, DNS:ip-172-31-24-120

Fixes #44

**Comment by lazypower** [comment]
Interesting, does this get applied to all server certs or only the cert generated on the leader?

**Comment by mbruzek** [comment]
Only on the cert generated by the leader at this time. The other certs will not get the extra SANs. I could change that if you think it would be useful.

File: reactive/k8s.py [hunk 1]
```diff
@@ -346,6 +359,14 @@ def get_dns_ip(cidr):
     return '.'.join(ip.split('.')[0:-1]) + '.10'
 
 
+def get_sdn_ip(cidr):
```
Comment (lazypower) [inline reactive/k8s.py:362]: Mark my words, these will bite us eventually, but until then we shall dance madly on the lip of an active volcano.

**Comment by lazypower** [comment]
I am +1 to this approach in its current form. I see why this was done the way it was implemented. We're working in a narrow hallway, and I think this satisfies the setup today.

**Pull request merged**